docs/pages/developer-guide.md

```diff
@@ -64,28 +64,18 @@ They store the desired configuration of a service and all relevant information r
 
 ## Creating a new standalone framework
 
-The `mesosphere/dcos-commons:latest` Docker image is able to set up a development environment for creating a new standalone framework.
+The `mesosphere/dcos-commons:latest` Docker image contains a development environment capable of creating new standalone frameworks.
 
-In order to set up this environment run the following command:
+In order to set up this environment run the following commands:
 ```bash
-$ FRAMEWORK_ROOT=/home/elezar/frameworks
+$ docker pull mesosphere/dcos-commons:latest # make sure you have the latest image
```

Comment:
I would split these two so that we don't have to rely on `#` comments in the bash command.

Reply:
Ack.